cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -149,6 +172,17 @@
             {% endif %}
         {% endfor %}
 
+        {% if is_share_view and is_printable %}
+            <li class="m-list_item m-social-media_print">
+                <button class="a-btn a-btn__link m-social-media_icon"
+                   onclick="{{ print_info.share_url }}"
+                   aria-label="{{ print_info.name }}"
+                   type="button">
```

Comment:
I thought so, too, but MDN convinced me otherwise. From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#notes:

>If your buttons are not for submitting form data to a server, be sure to set their type attribute to button. Otherwise they will try to submit form data and to load the (nonexistent) response, possibly destroying the current state of the document.

Seems like that should only be true if the button is tied to a form, but that note made it sound more general and a good idea for any button. I've never used that before and haven't seen any issues, but I guess it can't hurt?

Reply:
Ah, cool :+1: Good lookin' out.